src/table/utils/__tests__/handle-scroll.spec.js

```diff
@@ -159,12 +164,12 @@ describe('handle-scroll', () => {
         },
       };
       // targetOffsetTop = tableContainer.current.scrollTop - cell.offsetHeight - tableHead.offsetHeight;
-      // 700 - 100 - 128 = 472 => so our scrollTo function migth be called with 600
+      // 700 - 100 - 128 = 472 => so our scrollTo function might be called with 472
```

Comment:
Might be called? I mean it should be called right? otherwise it is not really a test

Reply:
I think you are right, let me fix it

Reply:
Done